.vscode/targets.log

```diff
@@ -0,0 +1,349 @@
+make all --print-data-base --no-builtin-variables --no-builtin-rules --question
+# GNU Make 3.81
```

Comment:
Did you intend to add these files?

Reply:
Not at all, it was a mistake. I'll add `.vscode` to gitignore